azure/datalake/store/multithread.py

```diff
@@ -245,7 +245,7 @@ def get_chunk(adlfs, src, dst, offset, size, buffersize, blocksize,
     for i in range(retries):
         try:
             nbytes = 0
-            with closing(_fetch_range(adlfs.azure, src, start=offset,
+            with closing(_fetch_range_with_retry(adlfs.azure, src, start=offset,
```

Comment:
Actually, you already in a retries loop, so now you retry inside of a retry (and this cannot be configured). It's what you want?